ballerina-tests/http-dispatching-tests/tests/service_dispatching_query_param_binding_test.bal

```diff
@@ -322,11 +343,11 @@ function testMapJsonOfDefaultableQueryBinding() returns error? {
 function testMapJsonArrOfDefaultableQueryBinding() returns error? {
     json response = check queryBindingClient->get("/default/q4");
     common:assertJsonPayloadtoJsonString(response, {
-        objects: [
-            {name: "test1", value: "json1"},
-            {name: "test2", value: "json2"}
-        ]
-    });
+                                                       objects: [
+                                                           {name: "test1", value: "json1"},
+                                                           {name: "test2", value: "json2"}
+                                                       ]
+                                                   });
```

Comment:
Is this intentional? for me, the previous formatting looks better.

Reply:
This was formatted using the `bal format` command and was not intentionally changed; it was kept as it was returned by the command. Do we need to revert it to the previous format?

Reply:
Address via [e3855c5](https://github.com/ballerina-platform/module-ballerina-http/pull/2152/commits/e3855c53a29907ccda8f2ae93758e028c99b6922)